component-runtime-beam/src/main/java/org/talend/sdk/component/runtime/beam/spi/record/AvroRecord.java

```diff
@@ -129,6 +133,13 @@ public Schema getSchema() {
         return schema;
     }
 
+    @Override
+    public Builder withNewSchema(final Schema schema) {
+        final AvroRecordBuilder builder = new AvroRecordBuilder(schema);
+        schema.getEntriesOrdered().forEach(e -> builder.with(e, delegate.get(9)));
```

Comment:
`get(9)` => error typing
With rewrite of RecordImpl, does not need override here (see below)

Reply:
Oups, yep!
I'm not sure to understand what you mean w/ RecordImpl and overriding?

Reply:
My bad, AvroRecord does not extends RecordImpl ...